hooks/HDG.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+/**
+ * Copyright 2016 Signal K and Joachim Bakke.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const debug = require('debug')('signalk-parser-nmea0183/HDG')
```

Comment:
You don't really need `debug` if you're not using it (it's a logger that only logs to `stdout` when the program is executed using `DEBUG=signalk* ...`)